web/pages/[[...slug]].tsx

```diff
@@ -150,6 +164,13 @@ const Route = ({
 
   return (
     <>
+      <NextSeo
+        title={title}
+        description={seoDescription}
+        canonical={urlJoin('https://structuredcontent.live', currentPath)}
+        noindex={noIndex}
+        openGraph={image ? { images: [{ url: image?.asset?.url }] } : undefined}
```

Comment:
I tried to test this with your shared-sections-test route, it doesn't seem to work: `<meta property="og:image"/>` with "View Source" on https://structured-content-2022-web-1xujtju6z.sanity.build/shared-sections-test.

Nit: I'm not sure, but I am guessing that the `.url` will just give a URL to the original image, however huge it may be. Ideally, I would think that it ought to be [transformed](https://www.sanity.io/docs/image-urls) in some way to fit or clip it within some limit. E.g. [Facebook requires](https://developers.facebook.com/docs/sharing/webmasters/images/) the image to be max 8 MB. 1260x630 might be a good max size…

Reply:
Good catch, I didn't think to add a test image to the test route.. 29411fc should address these concerns.